### PR TITLE
Jormungandr: use serpy for the schedule APIs

### PR DESCRIPTION
**Pull request by TeXitoi**
Use serpy for the schedule APIs:
  * [x] departures
  * [x] arrivals
  * [x] stop_schedules
  * [x] route_schedules

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Schedules.py [hunk 1]
```diff
@@ -231,7 +233,7 @@ class RouteSchedules(Schedules):
     def __init__(self):
         super(RouteSchedules, self).__init__("route_schedules")
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Schedules.py]: you need to define here `output_type_serializer`

and isn't it possible to use not `@get_serializer` but `get_obj_serializer` ?

File: source/jormungandr/jormungandr/interfaces/v1/serializer/schedule.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    uris = display_info.uris
+    l = [("line", uris.line),
+         ("company", uris.company),
+         ("vehicle_journey", uris.vehicle_journey),
+         ("route", uris.route),
+         ("commercial_mode", uris.commercial_mode),
+         ("physical_mode", uris.physical_mode),
+         ("network", uris.network),
+         ("note", uris.note)]
+    return [{"type": v[0], "id": v[1]} for v in l if v[1] != ""] + \
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/schedule.py]: minor: I think you can deconstruct the pair in the loop

File: source/jormungandr/jormungandr/interfaces/v1/serializer/schedule.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        if obj.time == __date_time_null_value__:
+            return ""
+        if obj.HasField('date'):
+            return timestamp_to_str(obj.date + obj.time)
+        return datetime.utcfromtimestamp(obj.time).strftime('%H%M%S')
+
+    def get_links(self, obj):
+        properties = obj.properties
+        r = []
+        #Note: all those links should be created with crete_{internal|external}_links,
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/schedule.py]: typo on `crete`

**Review by antoine-de — APPROVED** [review]

**Comment by TeXitoi** [comment]
Artemis approved: https://ci.navitia.io/view/artemis/job/Artemis_one_test/315/

**Review by antoine-de — APPROVED** [review]

**Pull request merged**